<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QPushButton" name="vote_button">
     <property name="text">
      <string>Проголосовать</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="cancel_button">
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Открытый ключ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QVBoxLayout" name="vars"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="vote_text_label">
     <property name="text">
      <string>Текст голосования</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="vote_name_label">
     <property name="text">
      <string>Название голосования</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="pubkey_lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="create_wallet_button">
     <property name="text">
      <string>Создать кошелек</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
